Monika After Story/game/script-compliments.rpy

```diff
@@ -46,11 +46,17 @@ label mas_compliments_start:
         # Unlock any compliments that need to be unlocked
         Event.checkConditionals(mas_compliments.compliment_database)
```

Comment:
Since we only had one compliment working with conditionals we could delete this line now, unless we have plans for using conditionals with these(I honestly don't think so).